lib/lib.go

```diff
@@ -32,7 +32,7 @@ func (v *Version) String() string {
 	} else if v.Major >= 0 {
 		return fmt.Sprintf("%d", v.Major)
 	} else {
-		return fmt.Sprintf("%!s(INVALID_VERSION)", v.Major)
+		return "%!s(INVALID_VERSION)"
```

Comment:
is this some kind of magic? what does `!s()` do?

Reply:
It's just what you get back from `fmt.Sprintf` and friends when there's an invalid format string.